### PR TITLE
[tflite2circle] Revise interface to pass raw file data

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise CircleModel interface to pass raw file data
to access Buffer data outside of flatbuffers range.

### CODE REVIEW

**Comment by seanshpark** [comment]
- for  #13673
- from #13615

File: compiler/tflite2circle/src/CircleModel.cpp [hunk 1]
```diff
@@ -376,8 +376,8 @@ template <> void Offset<OperatorCodeLink>::build(const TFLFlatBufVec *tflite_fla
   _circle_flatbuffer_vec_offset = _fb->CreateVector(operator_code_vec);
 }
 
-CircleModel::CircleModel(FlatBufBuilder &fb)
-  : _version{0}, _description{fb->CreateString("ONE-tflite2circle")}, _fb{fb}
+CircleModel::CircleModel(FlatBufBuilder &fb, const std::vector<char> &fr)
+  : _version{0}, _description{fb->CreateString("ONE-tflite2circle")}, _fb{fb}, _file_raw{fr}
```
Comment (mhs4670go) [inline compiler/tflite2circle/src/CircleModel.cpp:380]: I'm not sure I understand about this correctly. When this `_file_raw` is used?

Comment (mhs4670go) [reply compiler/tflite2circle/src/CircleModel.cpp:380]: Ah, I found the answer with the draft.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**